test/integration/apiserver/print_test.go

```diff
@@ -214,6 +218,20 @@ func TestServerSidePrint(t *testing.T) {
 			continue
 		}
 
+		obj, err := legacyscheme.Scheme.New(gvk)
+		if err != nil {
+			t.Errorf("unexpected error creating %s: %v", gvk, err)
+			continue
+		}
+
+		// if the API is removed in our level of kube or before, skip the type
+		if removedIn, ok := obj.(removedInterface); ok {
+			_, minor := removedIn.APILifecycleRemoved()
+			if minor <= 22 {
```

Comment:
switch this to something not hard-coded before merge?

Reply:
> switch this to something not hard-coded before merge?

where can I pull it from?

Reply:
Would it make more sense to flip this test to enable `api/all`, and loop through everything found via discovery to make sure it defines print handlers, rather than starting from things registered into the scheme? That would remove the need for `unservedTypes`, and for detecting `APILifecycleRemoved`, and just make sure whatever types can be served define handlers correctly.

Reply:
In a blue sky world, sure.  For this PR I can simply pull this update and see what else comes.